src/Controller/TagCloudController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Controller;
+
+use App\Repository\SearchesRepository;
+use Psr\Cache\InvalidArgumentException;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class TagCloudController extends AbstractController
+{
+    /**
+     * Returns TagCloud render with cached Top Tags.
+     *
+     * @param SearchesRepository $repository
+     * @return Response
+     * @throws InvalidArgumentException
+     */
+    #[Route('/TagCloud', name: 'TagCloud')]
+    public function getTopTagsPage(SearchesRepository $repository): Response
+    {
+        $topTags = $repository->getTopTags();
+
+        return $this->render('TagCloud.php.twig', [
```

Comment:
use snake case for twig filenames (https://symfony.com/doc/current/templates.html#template-naming)